appsec-kit-demo-v8/pom.xml

```diff
@@ -20,7 +20,7 @@
 	<properties>
 		<vaadin.version>8.8.0</vaadin.version>
 		<vaadin.plugin.version>8.8.0</vaadin.plugin.version>
-		<jetty.plugin.version>9.4.9.v20180320</jetty.plugin.version>
+		<jetty.plugin.version>10.0.15</jetty.plugin.version>
```

Comment:
With this new version there is an error when starting the v8 demo, after reverting back the app starts without error.
`
[WARNING] Error injecting: org.eclipse.jetty.maven.plugin.JettyRunMojo
java.lang.TypeNotPresentException: Type org.eclipse.jetty.maven.plugin.JettyRunMojo not present
...
Caused by: java.lang.UnsupportedClassVersionError: org/eclipse/jetty/maven/plugin/JettyRunMojo has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
`